ref/general/securing cloud native microservices.adoc

```diff
@@ -12,34 +12,33 @@
 :page-type: general
 = Securing cloud native microservices
 
-You can configure cloud native applications in Open Liberty to use external security providers for authentication and authorization.
+Cloud native applications are mostly microservices-based applications in a containerized environment. The microservices communicate among themselves and with other services. Applications run on different infrastructures with different security models. Any weak interaction in this complex environment is a cause for trouble. Developers need to make sure that their applications are secure. You can configure cloud native applications in Open Liberty to use external security providers for authentication and authorization.
 
-Authentication verifies the user identity based on user credentials while authorization determines user access to restricted resources.
-
-Jakarta EE Security configures authentication and authorization with annotations in servlets and provides `SecurityContext` API for programmatic security.
+https://draft-openlibertyio.mybluemix.net/docs/ref/general/#authentication.html[Authentication] verifies the user identity based on user credentials while https://draft-openlibertyio.mybluemix.net/docs/ref/general/#authorization.html[authorization] determines user access to restricted resources. Jakarta EE Security configures authentication and authorization with annotations in servlets and provides `SecurityContext` API for programmatic security.
 
 == Authentication
 
-Authentication verifies the user identity. Credentials are gathered. Cache is checked to see if any user subject is available. If no subject is found, then the Java Authentication and Authorization Service (JAAS) is called. The JAAS calls a set of login modules. One or more login modules create the subject based on credentials. The login module then calls the configured user registry for validation of credentials. Upon successful validation, relevant user information is collected and stored in the subject as credentials in the form of an SSO token. After successful authentication, the SSO is sent back to the browser in a cookie.
-The default name of the configurable cookie is ltpaToken2.
+Applications in the cloud deal with a number of users and services who try to access the application resources.
+Authentication verifies identity, where the user or service credentials are checked against a database for verification purposes.
 
-On subsequent calls, the token information is used to authenticate the user. If this authentication fails, the authentication service tries to use other authentication data, such as the user ID and password, if they still exist in the request.
+Authentication happens at the start of the application. The most common kind of authentication is username and password. A request to access is verified against the user registry and the information
```

Comment:
Change "happens at the start of the application" to "happens when the application starts." This also can help reduce wordiness.
Change "The most common kind of authentication is username and password" to "The most common kind of authentication is the username and password method." I think the original phrasing here could use a noun here to add more clarity.